.travis.yml

```diff
@@ -59,26 +59,40 @@ matrix:
           compiler: gcc
 
 before_script:
-    - sh .travis-create-release.sh $TRAVIS_OS_NAME
-    - tar -xvzf _srcdist.tar.gz
-    - mkdir _build;
-    - cd _build;
+    - if [ -n "$DESTDIR" ]; then
+          srcdir=../_srcdist;
+          top=..;
+          sh .travis-create-release.sh $TRAVIS_OS_NAME;
+          tar -xvzf _srcdist.tar.gz;
+          mkdir _build;
+          cd _build;
+      else
+          srcdir=.
+          top=.
+      fi
     - if [ "$CC" == i686-w64-mingw32-gcc ]; then
           export CROSS_COMPILE=${CC%%gcc}; unset CC;
-          ../_srcdist/Configure mingw $CONFIG_OPTS -Wno-pedantic-ms-format;
+          $srcdir/Configure mingw $CONFIG_OPTS -Wno-pedantic-ms-format;
       elif [ "$CC" == x86_64-w64-mingw32-gcc ]; then
           export CROSS_COMPILE=${CC%%gcc}; unset CC;
-          ../_srcdist/Configure mingw64 $CONFIG_OPTS -Wno-pedantic-ms-format;
+          $srcdir/Configure mingw64 $CONFIG_OPTS -Wno-pedantic-ms-format;
       else
           if which ccache >/dev/null && [ "$CC" != clang-3.6 ]; then
               CC="ccache $CC";
           fi;
-          ../_srcdist/config $CONFIG_OPTS;
+          $srcdir/config $CONFIG_OPTS;
       fi
-    - cd ..
+    - cd $top
 
 script:
-    - cd _build;
+    - if [ -n "$DESTDIR" ]; then
+          srcdir=../_srcdist;
```

Comment:
You're not using srcdir here I think? Also, as above, I'd move `top=..;` after `cd _build;`.